voter/src/main/java/org/terracotta/voter/ActiveVoter.java

```diff
@@ -96,6 +96,7 @@ public ActiveVoter start() {
 
   private Thread voterThread(CompletableFuture<VoterStatus> voterStatus, Optional<Properties> connectionProps, String... hostPorts) {
     return new Thread(() -> {
+      writePID();
```

Comment:
I think its better to dump PID in TCVoterMain as it is useful for script invocations mostly.